enterprise/cypher/cypher/src/test/scala/org/neo4j/cypher/internal/compatibility/v3_3/runtime/interpreted/RegisteredPipeBuilderTest.scala

```diff
@@ -305,7 +305,7 @@ class RegisteredPipeBuilderTest extends CypherFunSuite with LogicalPlanningTestS
     )())
   }
 
-  test("aggregation used for distinct") {
+  ignore("aggregation used for distinct") {
```

Comment:
Are we ignoring these because we are in an intermediate state and only want to fix the tests once some additional work is done? Would be nice for a TODO here stating what work still needs to be done. I'm guessing the TODO is aggregation?